src/frontend/context/OppgaverContext.tsx

```diff
@@ -190,14 +190,7 @@ const [OppgaverProvider, useOppgaver] = createUseContext(() => {
         settOppgaveFelter(initialOppgaveFelter(innloggetSaksbehandler));
     };
 
-    const { opprettEllerHentFagsak } = useFagsakApi(
```

Comment:
Denne koden gjør null og niks for å håndtere feilmeldinger. Jeg sletter den og setter på TODOs i stedet, det er bedre med TODOs enn kode som ikke gjør noe som helst. På lang sikt ønsker vi oss Toast-funksjonalitet på samme måte som i BA her.

Reply:
Vil du opprette teknisk gjeld-kort på toasts hvis det ikke allerede finnes? 🙏